rpc/grpc/types.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-gogo. DO NOT EDIT.
 // source: rpc/grpc/types.proto
 
-package core_grpc
```

Comment:
this file is auto-generated, maybe within the golangci configuration we ignore all  .pb.go files

Reply:
Yes understood, but the tests fail without this change. I can revert but then this file should be regenerated. I'm not sure how to regenerate proto buff files.